services/QuillLMS/client/app/bundles/Diagnostic/components/feedback/concept-feedback.jsx

```diff
@@ -10,7 +10,9 @@ class ConceptFeedback extends React.Component {
     const { dispatch, match } = this.props;
     const { params } = match;
     const { conceptFeedbackID } = params;
-    dispatch(actions.deleteConceptsFeedback(conceptFeedbackID))
+    if (confirm('⚠️ Are you sure you’d like to delete this concept feedback?')) {
+      dispatch(actions.deleteConceptsFeedback(conceptFeedbackID, () => window.location.reload()))
```

Comment:
Is there a reason that we're doing a `reload` callback for Diagnostic, but not Connect or Grammar?  Not a problem or anything, but I wanted to understand the discrepancy.

Reply:
whoops, this is outdated (from an earlier implementation that I decided not to go with). I removed this!